Engineering/SourceControl.md

```diff
@@ -1,5 +1,5 @@
 # Source Control
-There are many different options when working with Source Control. In [CSE](../CSE.md) we use [VSTS](https://csesd.visualstudio.com/_projects) for private repositories and [GitHub](https://github.com/) for public repositories.
+There are many different options when working with Source Control. In [CSE](../CSE.md) we use [Azure DevOps](https://csesd.visualstudio.com/_projects) for private repositories and [GitHub](https://github.com/) for public repositories.
```

Comment:
- Consider lowercasing "Source Control."
- Consider removing the link for Azure DevOps, as this project should not be externally accessible. Or, change to something like https://azure.microsoft.com/en-us/services/devops/, which is a landing page for describing the features of Azure DevOps.

Reply:
Hello Rachel, updated the pull request. Thank you for your feedback